Add to `tctl workflow reset` reference (#1576)
temporalio/documentation: docs/tctl/workflow/reset.md

`tctl workflow reset <modifiers>`

> The modifiers control the behavior of the command.

Reviewer: ```suggestion
```

### `--event_id`

Specify the `eventId` number of the event to which you want to reset.

> The event type must be one of: `WorkflowTaskCompleted`, `WorkflowTaskFailed`, and `WorkflowTaskTimeout`.

Author: Clarifying that `WorkflowTaskCompleted` is not a valid value for `--event_id`. Must be a number.

> Specify the `eventId` number of the event to which you want to reset.
> The event type must be one of: `WorkflowTaskCompleted`, `WorkflowTaskFailed`, and `WorkflowTaskTimeout`.

Reviewer: ```suggestion
Specify the `eventId` of the event to which you want to reset.
The event type must be one of the following: `WorkflowTaskCompleted`, `WorkflowTaskFailed`, or `WorkflowTaskTimeout`.
```
